src/wallet/wallet.h

```diff
@@ -494,6 +500,7 @@ class CWalletTx : public CMerkleTx
     CAmount GetCredit(const isminefilter& filter) const;
     CAmount GetImmatureCredit(bool fUseCache=true) const;
     CAmount GetAvailableCredit(bool fUseCache=true) const;
+    CAmount GetLockedCredit() const;
```

Comment:
what's this for?

Reply:
It's for getting locked balance when shielding coin as @egyptianbman said